x-pack/plugins/ml/public/application/components/rule_editor/condition_expression.js

```diff
@@ -102,7 +99,7 @@ export class ConditionExpression extends Component {
 
   renderAppliesToPopover() {
     return (
-      <div style={POPOVER_STYLE}>
+      <div>
```

Comment:
As discussed, the font size for the 'update condition' part of the related flyout looks too big. Could we drop that down as part of this fix (16px to 14px?).

<img width="437" alt="image" src="https://user-images.githubusercontent.com/7405507/191723877-cef080d1-1a81-4112-8704-d5532cfa5673.png">

Reply:
updated in https://github.com/elastic/kibana/pull/141215/commits/959e5b62315a2382e6c42e4b684bff47e508eba6